components/webui/imports/ui/SearchView/SearchControls.jsx

```diff
@@ -164,24 +172,23 @@ const SearchFilterControlsDrawer = ({
                         <SearchControlsDatePicker
                             id="begin_timestamp_picker"
                             selectsStart={true}
-                            startDate={timeRange.begin}
-                            endDate={timeRange.end}
-                            selected={timeRange.begin}
+                            startDate={convertUtcToSameLocalDate(timeRange.begin)}
```

Comment:
Since the lack of UTC support is in the react-date-picker library, the current changes saves UTC times and do the conversions at the date pickers' interface-level, instead of storing converted times. Feel free to object.